syncd/syncd_saiswitch.cpp

```diff
@@ -500,20 +494,16 @@ void SaiSwitch::removeExistingObject(
     }
 }
 
-std::vector<sai_port_stat_t> SaiSwitch::saiGetSupportedCounters() const
+void SaiSwitch::saiInitSupportedPortCounters()
```

Comment:
NO!. all saiXXX functions are just call to sai, and should be const, and should return queries value, bring that back!